mask_dsp_l.db file seems to be invalid

The `mask_XXX.db` files should be in the format at;
```
::::::::::::::
database/artix7/mask_bram_l.block_ram.db
::::::::::::::
bit 00_00
bit 00_01
bit 00_02
bit 00_03
bit 00_04
bit 00_05
bit 00_06
bit 00_07
bit 00_08
bit 00_09
bit 00_10
bit 00_11
bit 00_12
bit 00_13
bit 00_14
bit 00_15
bit 00_16
bit 00_17
bit 00_18
bit 00_19
bit 00_20
bit 00_21
bit 00_22
bit 00_23
bit 00_24
...
::::::::::::::
database/artix7/mask_clbll_l.db
::::::::::::::
bit 00_00
bit 00_01
bit 00_02
bit 00_03
bit 00_04
bit 00_05
bit 00_06
bit 00_07
bit 00_08
bit 00_09
bit 00_10
...
```

The `dsp_mask_l.db` file seems to have output like the following;
```
DSP.DSP48.DSP_0.MASK[0] 27_01
DSP.DSP48.DSP_0.MASK[10] 27_26
DSP.DSP48.DSP_0.MASK[11] 26_28
DSP.DSP48.DSP_0.MASK[12] 26_41
```

After doing a clean build, when running `make formatdb` you get the following;
```
Sorting DB   file mask_dsp_l.db                            Traceback (most recent call last):
  File "../../utils/sort_db.py", line 363, in <module>
    sys.exit(main(sys.argv))
  File "../../utils/sort_db.py", line 347, in main
    x = sort_db(n)
  File "../../utils/sort_db.py", line 303, in sort_db
    tosort.append(sortable_line_from_dbfile(l))
  File "../../utils/sort_db.py", line 233, in sortable_line_from_mask
    assert tag == 'bit', tag
AssertionError: DSP.DSP48.DSP_0.MASK[0]
Makefile:75: recipe for target 'formatdb' failed
make: *** [formatdb] Error 1
```

I rerun the fuzzers at commit 931576a770a6dec5ca91c8d3368be59a98c19817 and am still getting this behaviour. 
```
(env) [UPDATE PRJXRAY DATABASE] tansell@instance-1:~/github/SymbiFlow/prjxray$ make formatdb

Formatting database/artix7                                   
============================
Ignoring    file element_counts.csv                       .. success.
Sorting DB   file mask_bram_l.block_ram.db                 .. success.
Sorting DB   file mask_bram_l.db                           .. success.
Sorting DB   file mask_bram_r.block_ram.db                 .. success.
Sorting DB   file mask_bram_r.db                           .. success.
Sorting DB   file mask_clbll_l.db                          .. success.
Sorting DB   file mask_clbll_r.db                          .. success.
Sorting DB   file mask_clblm_l.db                          .. success.
Sorting DB   file mask_clblm_r.db                          .. success.     
Sorting DB   file mask_dsp_l.db                            .. success.
Sorting DB   file mask_dsp_r.db                            Traceback (most recent call last):
  File "../../utils/sort_db.py", line 363, in <module>          
    sys.exit(main(sys.argv))          
  File "../../utils/sort_db.py", line 347, in main
    x = sort_db(n)                                                                   
  File "../../utils/sort_db.py", line 303, in sort_db
    tosort.append(sortable_line_from_dbfile(l))                            
  File "../../utils/sort_db.py", line 233, in sortable_line_from_mask
    assert tag == 'bit', tag                   
AssertionError: DSP.DSP48.DSP_0.MASK[0]
Makefile:75: recipe for target 'formatdb' failed           
make: *** [formatdb] Error 1       
```

`mask_dsp_r.db` seems to have both `bit XXX` lines **and** `DSP.DSP48.DSP_0.PATTERN[0] 26_01` lines.

--- a/fuzzers/100-dsp-mskpat/Makefile
+++ b/fuzzers/100-dsp-mskpat/Makefile
@@ -10,14 +10,14 @@ SEGDATAS_R=$(addsuffix /segdata_dsp_r.txt,$(SPECIMENS))
 build/segbits_dsp_l.rdb: $(SPECIMENS_OK)
 	${XRAY_SEGMATCH} -o build/segbits_dsp_l.rdb $(SEGDATAS_L)
 
-build/segbits_dsp_l.db build/mask_dsp_l.db: build/segbits_dsp_l.rdb
+build/segbits_dsp_l.db: build/segbits_dsp_l.rdb
 	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --seg-fn-in $^ --seg-fn-out $@
 	${XRAY_MASKMERGE} build/mask_dsp_l.db $(SEGDATAS_L)
 
 build/segbits_dsp_r.rdb: $(SPECIMENS_OK)
 	${XRAY_SEGMATCH} -o build/segbits_dsp_r.rdb $(SEGDATAS_R)
 
-build/segbits_dsp_r.db build/mask_dsp_r.db: build/segbits_dsp_r.rdb
+build/segbits_dsp_r.db: build/segbits_dsp_r.rdb
 	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --seg-fn-in $^ --seg-fn-out $@
 	${XRAY_MASKMERGE} build/mask_dsp_r.db $(SEGDATAS_R)
 

